cpp/ql/test/library-tests/ir/ir/PrintAST.expected

```diff
@@ -18617,6 +18767,163 @@ struct_init.cpp:
+#    9| [MoveConstructor] void ClassWithConstructor::ClassWithConstructor(ClassWithConstructor&&)
+#    9|   <params>: 
+#-----|     getParameter(0): [Parameter] (unnamed parameter 0)
+#-----|         Type = [RValueReferenceType] ClassWithConstructor &&
+#   11| [Constructor] void ClassWithConstructor::ClassWithConstructor(char, char)
+#   11|   <params>: 
+#   11|     getParameter(0): [Parameter] x
+#   11|         Type = [PlainCharType] char
+#   11|     getParameter(1): [Parameter] y
+#   11|         Type = [PlainCharType] char
+#   14| [TopLevelFunction] char temp_test()
+#   14|   <params>: 
+#   14|   getEntryPoint(): [BlockStmt] { ... }
+#   15|     getStmt(0): [DeclStmt] declaration
+#   15|       getDeclarationEntry(0): [VariableDeclarationEntry] definition of x
+#   15|           Type = [PlainCharType] char
+#   15|         getVariable().getInitializer(): [Initializer] initializer for x
+#   15|           getExpr(): [FunctionCall] call to get_x
+#   15|               Type = [PlainCharType] char
+#   15|               ValueCategory = prvalue
+#   15|             getQualifier(): [ConstructorCall] call to ClassWithDestructor2
+#   15|                 Type = [VoidType] void
+#   15|                 ValueCategory = prvalue
+#   15|             getImplicitDestructorCall(0): [DestructorCall] call to ~ClassWithDestructor2
+#   15|                 Type = [VoidType] void
+#   15|                 ValueCategory = prvalue
+#   15|               getQualifier(): [ReuseExpr] reuse of temporary object
+#   15|                   Type = [Class] ClassWithDestructor2
+#   15|                   ValueCategory = prvalue(load)
+#   15|             getQualifier().getFullyConverted(): [TemporaryObjectExpr] temporary object
+#   15|                 Type = [Class] ClassWithDestructor2
+#   15|                 ValueCategory = prvalue(load)
+#   16|     getStmt(1): [DeclStmt] declaration
+#   16|       getDeclarationEntry(0): [VariableDeclarationEntry] definition of y
+#   16|           Type = [Class] ClassWithConstructor
+#   16|         getVariable().getInitializer(): [Initializer] initializer for y
+#   16|           getExpr(): [ConstructorCall] call to ClassWithConstructor
+#   16|               Type = [VoidType] void
+#   16|               ValueCategory = prvalue
+#   16|             getArgument(0): [CharLiteral] 97
+#   16|                 Type = [PlainCharType] char
+#   16|                 Value = [CharLiteral] 97
+#   16|                 ValueCategory = prvalue
+#   16|             getArgument(1): [FunctionCall] call to get_x
+#   16|                 Type = [PlainCharType] char
+#   16|                 ValueCategory = prvalue
+#   16|               getQualifier(): [ConstructorCall] call to ClassWithDestructor2
+#   16|                   Type = [VoidType] void
+#   16|                   ValueCategory = prvalue
+#   16|                 getImplicitDestructorCall(0): [DestructorCall] call to ~ClassWithDestructor2
```

Comment:
This seems to be attached one level too deep. Still need to investigate this.

Reply:
Two levels two deep, I think? I'd expect it to be attached directly to the `ConstructorCall`

Reply:
You were right. Two levels indeed. This is now fixed.